README.md

```diff
@@ -19,32 +19,32 @@ return el.toReact()
 // Yields: <div style='color: red;' className='box'></div>
 ```
 
-It supports a wide range of DOM operations and will fool most libraries but it isn't exhaustive (the full DOM API is ludicrously large). It supports enough to work with D3 but will require you to fork and add to the project if you encounter something that's missing.
+There's also a **mixin** with animation support:
 
-You can think of this as a bare bones [jsdom][] that's built to bridge the gap between the declarative React and the imperative JavaScript world. We just need to expand it as we go along since jsdom is a huge project that solves different problems.
+```javascript
+// inside componentWillMount
+var faux = this.connectFauxDOM('div','chart');
+d3.doingAnAdvancedAnimationFor3secs(faux);
+this.animateFauxDOM(3500); // duration + margin
 
-I'm trying to keep it light so as not to slow down your render function. I want efficient, declarative and stateless code, but I don't want to throw away previous tools to get there.
+// inside render
+return {this.state.chart};
+```
 
-## Limitations
 
-It's great for...
+React-faux-DOM supports a wide range of DOM operations and will fool most libraries but it isn't exhaustive (the full DOM API is ludicrously large). It supports enough to work with D3 but will require you to fork and add to the project if you encounter something that's missing.
 
- * Static D3 components or other such libraries (things like Backbone should work too!)
- * D3 components with simple state and event interaction, like tooltips on charts
- * D3 components such as progress bars that can be animated using [react-motion][], for example
+You can think of this as a bare bones [jsdom][] that's built to bridge the gap between the declarative React and the imperative JavaScript world. We just need to expand it as we go along since jsdom is a huge project that solves different problems.
 
-It's not so great for...
+I'm trying to keep it light so as not to slow down your render function. I want efficient, declarative and stateless code, but I don't want to throw away previous tools to get there.
 
- * Physics based D3 components or anything using a lot of DOM mutation and state
- * Linked to the previous one, brushing and filtering of selections using the built in stateful D3 tools
- * Essentially: Anything with a lot of DOM mutation from timers, events or internal state will be hard to use
 
-If you keep it stateless and React-ish then you'll be fine. Use tools like D3 to fluently build your charts / DOM, don't use it as an animation / physics / DOM mutation library, that doesn't work within React. See the state example linked below for a simple way to handle state, events and D3.
 
 ## Usage
 
  * Full [documentation][] with current DOM API coverage
  * [An example static chart ][lab-chart] ([source][lab-chart-source])
+ * [An example animated chart using the mixin](./examples/animate-d3-with-mixin)
```

Comment:
For consistency this should be `[An example animated chart using the mixin][mixin-example]` then add `[mixin-example]: ./examples/animate-d3-with-mixin` to the bottom of the document with the other links.